From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: OpenATV <openatv@openatv.org>
Date: Fri, 14 Feb 2026 00:00:00 +0000
Subject: [PATCH] LogBackend: replace fmt::localtime with std::localtime

fmt::localtime() was removed in fmt 11.0. Use *std::localtime()
directly instead, which produces the same std::tm value that fmt
can format with chrono specs.

---
 src/LogBackend.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/LogBackend.cxx b/src/LogBackend.cxx
--- a/src/LogBackend.cxx
+++ b/src/LogBackend.cxx
@@ -84,7 +84,7 @@ log_date() noexcept
 	static constexpr size_t LOG_DATE_BUF_SIZE = std::char_traits<char>::length("2025-01-22T15:43:14 ") + 1;
 	static char buf[LOG_DATE_BUF_SIZE];
 	time_t t = time(nullptr);
-	return FmtUnsafeSV(buf, "{:%FT%T} "sv, fmt::localtime(t));
+	return FmtUnsafeSV(buf, "{:%FT%T} "sv, *std::localtime(&t));
 }
 
 #ifdef HAVE_SYSLOG
